src/Display/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import 'jest-styled-components';
+import '@testing-library/jest-dom/extend-expect';
+import { fireEvent, render } from '@testing-library/react';
+
+import Tooltip from './Tooltip';
+
+describe('<Tooltip/> snapshots', () => {
+  const TooltipComponent = () => (
+    <Tooltip
+      text="mock text"
+      position="right"
```

Comment:
Can we test the other positions as well? It would raise the components line-coverage to 100%.

Reply:
ok